src/main/java/net/dv8tion/jda/webhook/WebhookClient.java

```diff
@@ -136,18 +135,14 @@ public String getUrl()
      * Sends the provided {@link java.io.File File} to this webhook.
      * <br>Use {@link WebhookMessage#files(String, Object, Object...)} to send up to 10 files!
      *
-     * @param  file
-     *         The file to send
-     *
-     * @throws java.lang.IllegalArgumentException
-     *         If the provided file is {@code null}, does not exist or is not readable
-     * @throws java.util.concurrent.RejectedExecutionException
-     *         If this client was closed
-     *
+     * @param file The file to send
      * @return {@link net.dv8tion.jda.core.requests.RequestFuture RequestFuture} representing the execution task,
-     *         this will be completed once the message was sent.
+     * this will be completed once the message was sent.
+     * If {@code isWait} is set to true this will have the message from discord, null otherwise
+     * @throws java.lang.IllegalArgumentException              If the provided file is {@code null}, does not exist or is not readable
+     * @throws java.util.concurrent.RejectedExecutionException If this client was closed
```

Comment:
This is not the format used in JDA, throws should be __before__ return. I advise to look at some other examples of this in JDA.